test/built-ins/Map/iterator-items-are-not-object.js

```diff
@@ -44,5 +44,7 @@ assert.throws(TypeError, function() {
 });
 
 assert.throws(TypeError, function() {
-  new Map([['a', 1], 2]);
+  new Map([
+    ['a', 1], 2
```

Comment:
This is weird; the 2 should be on its own line.

Reply:
lol... agreed. Machines can't (always?) be trusted.